lib/routes/sdu/cs.ts

```diff
@@ -25,14 +37,14 @@ export const route: Route = {
     name: '计算机科学与技术学院通知',
     maintainers: ['Ji4n1ng'],
     handler,
-    description: `| 学院公告 | 学术报告 | 科技简讯 |
-  | -------- | -------- | -------- |
-  | 0        | 1        | 2        |`,
+    description: `| 学院公告 | 学术报告 | 科技简讯 | 本科教育 | 研究生教育 | 
+                  | -------- | -------- | -------- | -------- | -------- |
+                  | announcement | academic | technology | undergraduate | postgraduate |`,
```

Comment:
Indent it with 0 or 2 spaces, or it won't be rendered properly/

Reply:
[1312c73](https://github.com/DIYgod/RSSHub/pull/16792/commits/1312c735ecadc21efaba81650ac5e5b50a89675a) solved